index.bs

```diff
@@ -290,12 +290,8 @@ Advertising agents must include DNS TXT records with the following
 keys and values:
 
 : fp
-:: The <dfn>agent fingerprint</dfn> of the advertising agent, computed over the
-    [=agent certificate=]. The format of the fingerprint is defined by [[!RFC5122]],
-    excluding the "fingerprint:" prefix and including the hash function, space,
-    and hex-encoded fingerprint.  The fingerprint value also functions as an ID
-    for the agent. All agents must support the following hash functions: [=sha-256=],
-    [=sha-512=].  Agents must not support the following hash functions: [=md2=], [=md5=].
+:: The [=agent fingerprint=] of the advertising agent. The steps to compute
+    the agent fingerprint are defined below.
```

Comment:
Since it's not defined immediately below, consider adding a section reference.

Reply:
The `[=agent fingerprint=]` syntax will link directly to the definition, do you think something else is needed?